collector/collector.go

```diff
@@ -476,12 +476,12 @@ func (c Config) Query(s string) (res string, err error) {
 	if err := json.Unmarshal(b, &dat); err != nil {
 		panic(err)
 	}
-
-	query, err := gojq.Parse(s)
+	// Adding "// empty" to the query so the output does not include a "null" if the value is empty
+	// This is not a json parse feature but a string one, so we should return normal values not json specific ones
```

Comment:
Is this `// empty` thing documented somewhere? I looked at the gojq code but I can't find anything.

Reply:
empty comes from upstream jq, which gojq implements: https://jqlang.github.io/jq/manual/#empty

And "//" is an alternative operator. It means that for "a // b" if a is empty it will return b: https://jqlang.github.io/jq/manual/#alternative-operator so in this case if the query results into a null value, we return empty instead :D

Reply:
Gotcha! I think it's good to add this link to the comments.

Reply:
ohhh nice to know